DenunciaCiudadana/src/store/store.js

```diff
@@ -0,0 +1,14 @@
+import {createStore, combineReducers, applyMiddleware} from 'redux';
+import {composeWithDevTools} from 'redux-devtools-extension';
+import thunk from 'redux-thunk';
+
+import {exampleReducer} from '../reducers/exampleReducers';
+
+const reducers = combineReducers({
+  example: exampleReducer,
+});
+
+export const store = createStore(
+  reducers,
+  composeWithDevTools(applyMiddleware(thunk)),
```

Comment:
For development, I recommend the use of redux logger and redux devtools
[ https://github.com/LogRocket/redux-logger](url)